woocommerce/woocommerce-ios
# [Shipping labels] Carriers list optimizations

## Description


- update carriers packages list header font
- used simple animation when changing starred list

## Testing information


- Install and set up the Woo Shipping extension on your store.
- Build and run the app with the revampedShippingLabelCreation feature flag enabled.
- Create an order with the processing status and at least one physical product.
- In the order details, select "Create Shipping Label."
- Tap "Select a Package" button
- Switch to "Carrier" tab
- Check the headers text font
- Tap to star and unstar the package and observe the slight animation of the icon

## Screenshots


https://github.com/user-attachments/assets/40c7f144-3d31-4217-96b9-cbb6d5e405e7

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
